latest/src/app/documentation/demos/buttons/utils/example.ts

```diff
@@ -4,7 +4,7 @@
  * The full license information can be found in LICENSE in the root directory of this project.
  */
 import {Component, Input, ViewChild, AfterViewInit} from "@angular/core";
-import {CodeHighlight} from "@clr/angular";
+import {CodeHighlight} from "../../../../utils/code-highlight";
```

Comment:
It seems like we have this, which duplicates the same code highlight functionality, and this appears to be used in a few random places. Can we use just the one implementation and remove this?